### PR TITLE
Improvements to type to support alembic

### PR DESCRIPTION
**Pull request by CaselIT**
Various changes to support typing of alembic.

~~still wip~~

### CODE REVIEW

**Comment by bryanforbes** [comment]
Looks good to me

**Comment by CaselIT** [comment]
@zzzeek could you take a look at this one? These change were done to make alembic pass

File: sqlalchemy-stubs/schema.pyi [hunk 1]
```diff
@@ -21,6 +21,7 @@ from .sql.ddl import SetTableComment as SetTableComment
 from .sql.ddl import sort_tables as sort_tables
 from .sql.ddl import sort_tables_and_constraints as sort_tables_and_constraints
 from .sql.naming import conv as conv
+from .sql.schema import _get_table_key as _get_table_key
```
Comment (zzzeek) [inline sqlalchemy-stubs/schema.pyi:24]: hmmm because alembic uses it?  OK

**Review by zzzeek — APPROVED** [review]
LGTM, of course someday not having underscore things and less Optional would be great but lets get the world all on Py3K first

File: sqlalchemy-stubs/schema.pyi (hunk 1 above)
Comment (CaselIT) [reply sqlalchemy-stubs/schema.pyi:24]: yes

**Comment by CaselIT** [comment]
thanks for the review. let's merge this and retest the alembic change

**Pull request merged**